modules/service/replication-service.js

```diff
@@ -44,47 +44,50 @@ class ReplicationService {
         if (!offer) {
             throw new Error(`Failed to find offer with internal ID ${internalOfferId}`);
         }
-
+        this.logger.info('getImport');
         const otJson = await this.importService.getImport(offer.data_set_id);
-
+        this.logger.info('getImport finalized');
         await this.permissionedDataService.addDataSellerForPermissionedData(
             offer.data_set_id,
             this.config.erc725Identity,
             this.config.default_data_price,
             this.config.identity,
             otJson['@graph'],
         );
-
+        this.logger.info('addDataSellerForPermissionedData');
         ImportUtilities.removeGraphPermissionedData(otJson['@graph']);
-
+        this.logger.info('removeGraphPermissionedData');
         const hashes = {};
 
         const writeFilePromises = [];
         this.replicationCache[internalOfferId] = {};
         for (let i = 0; i < 3; i += 1) {
             const color = this.castNumberToColor(i);
-
+            this.logger.info('generateKeyPair');
```

Comment:
We should refactor these logs or use debug level at least.

Reply:
These logs were just for testing purposes. I'll remove this whole method since we are not using it anymore